proposals/CSP.md

```diff
@@ -132,31 +132,39 @@ We recommend that a new CSP policy directive `wasm-unsafe-eval` be created. If s
 
 Given the current usage of the CSP policy `unsafe-eval` to gate both JavaScript `eval` and instantiating WebAssembly modules, we propose that that behavior be allowed to continue; but that `wasm-unsafe-eval` should have no implication for JavaScript loading or evaluation or use of `eval` in JavaScript. NOTE: Providing a directive to allow JavaScript `eval` without WebAssembly doesn't seem immediately useful, and so has been left out intentionally.
 
+### Using existing CSP script-src policies
+
+While `WebAssembly.compile` and `WebAssembly.instantiate` take arrays of bytes to produce executable code, `WebAssembly.compileStreaming` and `WebAssembly.intantiateStreaming` depend on a `Response` object. One characteristic of the `Response` object is that it has access to the URL of the original request.
+
+We recommend that this URL may be vetted by the `script-src` CSP policy associated with the _container_ (typically either an IFrame or the page executing the JavaScript call to `compileStreaming` or `instantiateStreaming` APIs). 
```

Comment:
When would it be associated with an iframe?

Reply:
removed reference to iframe as it is potentially confusing